src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java

```diff
@@ -1728,6 +1728,13 @@ void onPlayerInteract(PlayerInteractEvent event)
         }
 
         //apply rules for containers and crafting blocks
+        Set<Material> candles = new HashSet<>();
```

Comment:
There's a [tag for candles](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/Tag.html#CANDLES), no need to manually assemble a listing.